src/session/udev.c

```diff
@@ -200,10 +200,22 @@ input_event(int fd, uint32_t mask, void *data)
          break;
 
          case LIBINPUT_EVENT_TOUCH_UP:
+         {
+            struct libinput_event_touch *tev = libinput_event_get_touch_event(event);
+            struct wlc_input_event ev;
```

Comment:
Probably good idea to do the zero intialize for every input event here. But better do that as another commit/PR to keep focus.